interview-questions.md

```diff
@@ -7,44 +7,75 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: 
+        The foreign key would be on the Student model. by creating a new column and importing it. In order to migrate the foreign key either (not sure) an add_foreign keyn or add_column method can be used. Name i think it would be add_colum_student_name.
 
-  Researched answer:
+  Researched answer: 
+        A rails generate migration add_column_to students should be called in the terminal to add a column to the student model. The column added should have the name cohort_id
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: 
+        CRUD: Create, Read, Update, and Delete must always be passed params
 
+  Researched answer: 
+        A RESTful route is a route that provides a mapping of HTTP verbs (get, post, put, patch, delete) to the CRUD actions (create, read, update, delete) that are defined in our controllers.  The show, create, edit, update, and delete are required params for RESTFUL routes.
+        Show - shows an object
+        Create - makes a new object
+        Edit and Update - mackes changes to the object
+        Delete - deltess the object
```

Comment:
RESTful routes do map to CRUD actions but the questions is asking about params. There are some routes that require extra pieces of information. Usually because the action needs to happen on one particular item in the database - show, update, delete.